test/integration/answer_rendering_test.rb

```diff
@@ -26,7 +26,11 @@ class AnswerRenderingTest < ActionDispatch::IntegrationTest
         assert page.has_selector?(".modified-date", text: "Last updated: 2 October 2012")
       end
     end # within #content
-    assert page.has_selector?("#test-related")
+
+    # The following selectors are specified using XPath because Capybara/Nokogiri does not seem to find non-standard tags
+    # using the usual "CSS-style" selectors.
+    assert page.has_selector?(:xpath, "//test-govuk-component[@data-template='govuk_component-breadcrumbs']", visible: true)
```

Comment:
It would be nice to extract this into a test helper. Perhaps `assert_breadcrumb_rendered` and `assert_sidebar_rendered`?